infrastructure/azure.tf

```diff
@@ -840,6 +845,13 @@ resource "null_resource" "azurerm_apim_api" {
   }
 }
 
+# DNS Zone
+
+resource "azurerm_dns_zone" "azurerm_dns_main_zone" {
```

Comment:
you should make sure this is created only in the production environment, use the `count` trick in the pagopa module

Reply:
done. note that there are no issues running two dns zones with the same name (apart that only one will be used). problems arises when the zone is deleted and recreated as the azure ns hostnames will change and the NS record will invalidate.

Reply:
yes but it's not a correct implementation anyway, most importantly it would not communicate our intent :)

Reply:
fixed